src/gui/optionsdialog.ui

```diff
@@ -397,14 +397,35 @@
                 </widget>
                </item>
                <item>
-                <widget class="QCheckBox" name="checkStartMinimized">
-                 <property name="toolTip">
-                  <string>When qBittorrent is started, the main window will be minimized</string>
-                 </property>
-                 <property name="text">
-                  <string>Start qBittorrent minimized</string>
-                 </property>
-                </widget>
+                <layout class="QHBoxLayout" name="windowStateLayout">
+                 <item>
+                  <widget class="QLabel" name="windowStateLabel">
+                   <property name="text">
+                    <string>Initial window state:</string>
```

Comment:
Current one is good but maybe "Window state on start up"? Since a couple options above it use "start up". Also i think "initial" could be misunderstood as "a one time thing" i.e. a little confusing when seen for the first time.

Reply:
> maybe "Window state on start up"?

OK, changed.